client/src/controllers/PreviewController.test.js

```diff
@@ -425,6 +425,35 @@ describe('PreviewController', () => {
       );
     });
 
+    it('should have a fallback width if the input is missing the data attribute', async () => {
```

Comment:
Unrelated, just me obsessing over 100% test coverage in the whole `PreviewController` now that we incorporate the `runAccessibilityChecks` and `runContentChecks` into methods of the controller